src/app/dynamic-forms/fields/rich-text/rich-text.component.ts

```diff
@@ -80,35 +86,50 @@ export class RichTextComponent implements OnDestroy, AfterViewInit {
   activeMarks = {
     bold: false,
     code: false,
-    heading1: false,
-    heading2: false,
-    heading3: false,
-    heading4: false,
-    heading5: false,
+    codeBlock: false,
+    Heading1: false,
```

Comment:
Why were these capitalized?

Reply:
In the headings dropdown, I'm looping through `headings` where the array contains `{ value: "Heading1"},`.   Then in the html element I'm using `[selected]="activeMarks[heading.value]"` and `[value]="tiptapMarkType[heading.value]"`. Which uses the `value`, so I was thinking of avoiding to do `{ selected: 'heading1', value: 'Heading1'}` for this loop. Let me know if I should revert :P

Reply:
I think the better plan would be to keep this set consistent, and then in the headings array just set the value to "heading#" instead. Is there any reason why the value of the selected headings element would need to be capitalized?

Reply:
The reason the selected headings need to be capitalized is because on select change, it triggers `toggleMark` which requires `Heading#`.  Should I just add another key/value pair to keep this consistent then?

Reply:
Yeah, I don't like that. If we end up changing the structure of the enum then this will break and it will be really difficult to debug. I think in the headings array you should use something like `mark: TipTapMarkType.Heading#` and then use `[value]="heading.mark"` in the template. For the active flag on the list items, create a proxy function that takes in the mark type and returns the correct activeMark from a switch case. It's more verbose, for sure, but it's 100% clear and later will be much easier to unit test.